transform/views/test_views.py

```diff
@@ -0,0 +1,260 @@
+from transform.transformers import PDFTransformer, ImageTransformer, CSTransformer
+from transform import app
+from jinja2 import Environment, PackageLoader
+
+from flask import make_response, send_file
+import logging
+from structlog import wrap_logger
+import json
+import os.path
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+env = Environment(loader=PackageLoader('transform', 'templates'))
+
+test_message = """{
+   "type": "uk.gov.ons.edc.eq:surveyresponse",
+   "origin": "uk.gov.ons.edc.eq",
+   "survey_id": "144",
+   "version": "0.0.1",
+   "collection": {
+     "exercise_sid": "hfjdskf",
+     "instrument_id": "0001",
+     "period": "0216"
```

Comment:
Pedantic: is this meant to be 2016? I'm fine with it being a random number if it doesn't matter, but if we were aiming for 2016 so it looks like a real submission, we missed a little bit.